PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+** Describe the change **
+
+_explantation of what it does_
+
+** Issue reference **
+
+* If this is a fix for a JIRA, please put JIRA-reference in the PR title like _[KIALI-7] New icons_
```

Comment:
After voting for PR titles, we decided to put JIRA reference without the square brackets.

Reply:
Ok, good catch.